src/containers/ErrorPage/__tests__/__snapshots__/ErrorPage-test.js.snap

```diff
@@ -524,60 +524,58 @@ exports[`ErrorPage renderers correctly 1`] = `
   className="o-container o-container--backgroundWide"
 >
   <div
-    className="o-content"
+    className="c-masthead c-masthead--fixed"
+    id="masthead"
```

Comment:
Denne fikk en del nye klasser etter at du fjerna showLoaderWhenNeeded. Er det problematisk?

Reply:
Ser umiddelbart ganske likt ut for meg, men masthead er flyttet ut av `<Content>`, så ble derfor en litt kaotisk diff.